api/geth_backend.go

```diff
@@ -2118,6 +2120,7 @@ func (b *GethStatusBackend) startNode(config *params.NodeConfig) (err error) {
 	defer func() {
 		if r := recover(); r != nil {
 			err = fmt.Errorf("node crashed on start: %v", err)
+			sentry.RecoverError(err)
```

Comment:
@igor-sirotin, shouldn't this call be protected by a condition in case the user hasn't allowed the Sentry integration?

Reply:
@ilmotta, it shouldn't. By setting `SentryDSN = ""` we completely disable it.
https://github.com/status-im/status-go/blob/3969b8c7cca8f6cc0e4def7ac4f98de632ea86e7/internal/sentry/sentry.go#L26-L28

So this logic is encapsulated, Sentry internally will not send any events.
(I tested this myself).

Reply:
Okay, that's very convenient. So sentry calls will be no-ops automatically whenever the var is not set or empty?